app/views/result/_enhanced_results_table.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class="enhanced-results-table">
+  <% enhanced_results.each do |field, title_link_pair| %>
+    <tr>
+      <th class="e-table-title">
+        <%= field %>
+      </th>
+    </tr>
+      <% title_link_pair.each do |a| %>
```

Comment:
wrong indentation

also we can auto expand this out to have nicely named variables:

```
<% title_link_pair.each do |title, link| %>
```

Reply:
good point, done. For future refactors, is an array the right way to store this?

Reply:
Shrug, it is a bit of a code smell (Primitive Obsession) but it is also a simple way to get the data back. I wouldn't rush to refactor right now, but it might be worth thinking about alternatives and weighing them against each other next time you want to do something similar.